mobile/test/swift/integration/StatFlushIntegrationTest.swift

```diff
@@ -1,8 +1,14 @@
 import Envoy
 import Foundation
+import TestExtensions
 import XCTest
 
 final class StatFlushIntegrationTest: XCTestCase {
+  override static func setUp() {
+    super.setUp()
+    register_test_extensions()
```

Comment:
If this is too noisy, I can easily move this to a base test class that all Swift integration tests inherit from instead of `XCTestCase`.